CLAUDE.md

```
@@ -0,0 +1,66 @@
+# CLAUDE.md
+
+This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
+
+## Project Overview
+
+Docker wait for dependencies is a containerized tool that waits for services to be ready before proceeding. It supports:
+- **TCP connectivity checks**: Wait for services via `host:port` format
```

markdownlint-cli2 (0.18.1)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)